go.mod

```diff
@@ -324,6 +326,8 @@ replace github.com/shurcooL/vfsgen => github.com/lunny/vfsgen v0.0.0-20220105142
 
 replace github.com/nektos/act => gitea.com/gitea/act v0.259.1
 
+replace github.com/charmbracelet/git-lfs-transfer => gitea.com/gitea/git-lfs-transfer v0.2.0
```

Comment:
Can you send the changes to `gitea.com/gitea/git-lfs-transfer` as pull requests?

Reply:
What changes do you mean?

Reply:
The changes why you reference your fork rather than use Gitea's

Reply:
That is gitea's fork, though. You might be reading the directive backwards?

Reply:
I made a mistaken.